Client/Frontend/Library/Bookmarks/BookmarksPanel.swift

```diff
@@ -562,6 +649,60 @@ extension BookmarksPanel {
             presentInFolderActions()
         }
     }
+    
+    func showMoreDialog() {
```

Comment:
Just double checking whether the functions here can be made `private` 🙏

Reply:
Yep! Done.